### ISSUE TITLE
Trouble deploying site locally

### ISSUE THREAD

**Issue by MathewBiddle** [ioos/notebooks_demos#403]
In trying to debug site functionality, I've been having trouble deploying it locally on my machine. (I'm not sure if this should be a [ioos-jekyll-theme](https://github.com/ioos/ioos_jekyll_theme/tree/0eeb408412150d07fd2cb14c2a2e6d2bb3e8cf1d) concern or something with this site specifically.) I have been able to deploy other ioos.github.io sites locally without any issue (specifically the [ncei-archiving-cookbook](https://ioos.github.io/ncei-archiving-cookbook/). My troubles are listed below:

 - After installing **jekyll**, **ruby**, and **bundler**. I try to run the site locally (in `notebooks_demos\webpage`) using:
   ```bundle exec jekyll serve --config _config.yml,_config_dev.yml --watch --verbose --incremental```
   I get an error that the `Gemfile` or `.bundle/` directory can't be found.
- I can get it running using:
   ```jekyll serve --config _config.yml,_config_dev.yml --watch --verbose --incremental```
   But, I assume that's missing some of the build pieces which I might be trying to edit. 
   - Also, this doesn't allow me to look at the notebooks. When I click on Code Gallery, it returns an index: 
![image](https://user-images.githubusercontent.com/8480023/116102155-72604780-a67c-11eb-84ba-36874e4117e5.png)

To note: the `bundle exec` command works just fine with the ncei-archiving-cookbook site, when I deploy that locally. 

TLDR; Is there something different about this site where it doesn't follow the guidance documented [here](https://ioos.github.io/ioos-documentation-jekyll-skeleton/howto.html). Specifically building the site using bundler?

**Comment by mwengren** [comment]
@MathewBiddle Did you see any errors in the process logs?  I tried updating the IOOS Catalog [documentation site](https://ioos.githhub.io/catalog) (which also uses ioos_jekyll_theme) and before running locally, I updated by local Jekyll environment - see: https://jekyllrb.com/docs/upgrading/.  

Updating produced several deprecation warnings for some of the code inside the ioos_jekyll_theme module, which will probably require updating it to stay current - I'm thinking those issues could be contributing in your local setup perhaps.  

It's likely we'll need to look into updating the ioos_jekyll_theme regardless to keep it current.

**Comment by ocefpaf** [comment]
> It's likely we'll need to look into updating the ioos_jekyll_theme regardless to keep it current.

I have to confess that I have been postponing this until #386 became a reality and we would not need to do it :grimacing: 
I tried updating the theme a while back and found out it was easier to re-do the theme then updating it. Jekyll is fast, flexible, and easy. But not really good for upgrades once you deploy it.

**Comment by MathewBiddle** [comment]
I think #386 is a reality now. Closing.
